api/python/cellxgene_census/pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2"]
+requires = ["setuptools>=64", "setuptools_scm[toml]>=8"]
```

Comment:
This is only tangentially related to the new GHA, but important as the existing dependencies were quite obsolete and willl fail in some scenarios.  See https://setuptools-scm.readthedocs.io/en/latest/